logstash-8.yaml

```diff
@@ -171,6 +171,10 @@ subpackages:
   - name: ${{package.name}}-compat
     description: Compatibility with the upstream image
     dependencies:
+      runtime:
```

Comment:
These were defined as runtime dependencies at the image level. Moving to package level runtimes for consistency.